schema/me/conversation.js

```diff
@@ -37,25 +37,54 @@ export const BuyerOutcomeTypes = new GraphQLEnumType({
   },
 })
 
+export const AttachmentType = new GraphQLObjectType({
+  name: "AttachmentType",
+  desciption: "Fields of an attachment (currently from Radiation)",
+  fields: {
+    id: {
+      description: "Attachment id.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    content_type: {
```

Comment:
We'll be able to use `content_type` and `download_url` to offer PDF/Image previews.